soc/arm/nuvoton_npcx/npcx7/Kconfig.soc

```diff
@@ -3,17 +3,17 @@
 # Copyright (c) 2020 Nuvoton Technology Corporation.
 # SPDX-License-Identifier: Apache-2.0
 
-choice
-	prompt "NPCX7 Selection"
-	depends on SOC_SERIES_NPCX7
-
 config SOC_NPCX7M6FB
-	bool "NPCX7M6FB"
+	bool
+	help
+	  NPCX7M6FB
 
 config SOC_NPCX7M6FC
-	bool "NPCX7M6FC"
+	bool
+	help
+	  NPCX7M6FC
 
 config SOC_NPCX7M7FC
-	bool "NPCX7M7FC"
-
-endchoice
+	bool
+	help
+	  NPCX7M7FC
```

Comment:
i'm very much in favor of `help` texts, especially because they are rendered into the official documentation, but a help text which simply just adds the SoC name is not of much value, so for those I prefer to simply keep the config as-is, that is without a help text.

Reply:
that does not apply everywhere, `Nuvoton M48X Series MCU` is much more descriptibe than `SOC_SERIES_M48X`. This  commit is removing valuable information IMO.

Reply:
i'm only requesting not to add dummy help text where it is not existing.
(or improve the help text to be meaningful)

SoCs that has relevant help text should definitely keep that.

Reply:
The text was already there, in the selection field, this just moves it from the selection field (as it is no longer selectable) to the help field